app/app.go

```diff
@@ -1249,6 +1259,9 @@ func (app *Evmos) setupUpgradeHandlers() {
 		storeUpgrades = &storetypes.StoreUpgrades{
 			Added: []string{icahosttypes.SubModuleName, recoverytypes.StoreKey},
 		}
+	case v12.UpgradeName:
+		// ?? Should we rename here the recovery store
```

Comment:
@fedekunze  Q: Should we rename here the recovery store?

Reply:
Yeah we should

Reply:
Testnet has the correct name `recovery`

Reply:
> Testnet has the correct name recovery

ok, not sure if the upgrade panics when you try to rename a store that already has the same name